src/EnergyPlus/WaterThermalTanks.cc

```diff
@@ -6368,7 +6370,9 @@ namespace WaterThermalTanks {
 		if ( Tank.InletMode == InletModeFixed ) CalcNodeMassFlows( WaterThermalTankNum, InletModeFixed );
 
 		TimeRemaining = SecInTimeStep;
-		while ( TimeRemaining > 0.0 ) {
+		Real64 & tank_node1_newTemp( Tank.Node( 1 ).NewTemp );
+		Real64 const & tank_useOutletStratNode_temp( Tank.Node( Tank.UseOutletStratNode ).Temp );
+		while ( TimeRemaining > 0.0 ) { //? Might be good to use a check that is tolerant to some epsilon * dt to avoid an extra pass due to roundoff when subtracting dt (if dt is ever not a value with an exact floating point representation)
```

Comment:
@DeadParrot, I'll make sure to [address this](https://github.com/NREL/EnergyPlus/commit/9efa39dbb1a6b792575f7bcd0d876cc71a0c5dee#commitcomment-16065959) too. Thanks.